dev-python/Rx/Rx-3.1.0.ebuild

```diff
@@ -0,0 +1,13 @@
+EAPI=7
+
+PYTHON_COMPAT=( python3_6 python3_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="Reactive Extensions for Python"
+HOMEPAGE="http://reactivex.io/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
```

Comment:
Where are the tests?

Reply:
Not in the tarball.

Reply:
Use github archive.

Reply:
I can’t run the tests on 3.9 because `dev-python/pytest-asyncio` (which the package’s `setup.py` claims is needed; it’s in `tests_require`) is only in the tree for 3.8. Should I just test and declare up to 3.8? Or test up to 3.8 but declare 3.9 as well without testing? Or wait for a while and hope that `dev-python/pytest-asyncio` is updated quickly enough that I can still get this in before the package dies?

Reply:
Done

Reply:
Thanks; that was fast! Next question: a few of the tests need a `$DISPLAY`. Upstream, their Travis CI config runs pytest (via `setup.py test`) inside an `xvfb` instance. Is that an acceptable way to run tests for the Gentoo tree?

Reply:
See virtualx eclass

Reply:
9f3aa9e766ee